User/PasswordResetInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\User;
+
+interface PasswordResetInterface{
```

Comment:
we gebruiken over het algemeen geen `Interface` suffix. http://verraes.net/2013/09/sensible-interfaces/